src/tests/stdlib/text_replace.ab

```diff
@@ -1,8 +1,16 @@
 import * from "std/text"
 
 // Output
-// apple apple
-
+// TWO TWO TWO
+// a\\b\\c\\d
+// first..second..third..fourth
+// mono
+// di
+// tri
```

Comment:
Please add a blank line below, to separate the expected output from the `main` block.

Reply:
Done